decoder/weather.go

```diff
@@ -147,6 +148,7 @@ func createWeatherWebhooks(oldWeather *Weather, weather *Weather) {
 			"updated":              weather.Updated,
 		}
 		areas := MatchStatsGeofence(weather.Latitude, weather.Longitude)
+		external.UpdateWeatherCount(areas, weather.GameplayCondition.ValueOrZero())
```

Comment:
Same comment applies... or is this because you want the areas and so do it here?
In Pokemon we calculate the areas earlier because it is used for stats as well; perhaps it would be more appropriate to do that - ie calculate the areas in the save record, and use for prometheus and pass to the webhook?

Reply:
I will move it up. We have lat,lon on inserted data in this case.

Reply:
`Total number of weather updates by gameplay condition`
looks valid for:
`oldWeather == nil || oldWeather.GameplayCondition.ValueOrZero() != weather.GameplayCondition.ValueOrZero() || oldWeather.WarnWeather.ValueOrZero() != weather.WarnWeather.ValueOrZero()`

Reply:
dropped